.github/workflows/tests.yaml

```diff
@@ -98,11 +98,13 @@ jobs:
         run: |
           sudo mkdir -p /usr/src/showcase
           sudo chown -R ${USER} /usr/src/
-          curl --location https://github.com/googleapis/gapic-showcase/releases/download/v0.12.0/gapic-showcase-0.12.0-linux-amd64.tar.gz --output /usr/src/showcase/showcase-0.12.0-linux-amd64.tar.gz
+          curl --location https://github.com/googleapis/gapic-showcase/releases/download/v${SHOWCASE_VERSION}/gapic-showcase-${SHOWCASE_VERSION}-linux-amd64.tar.gz --output /usr/src/showcase/showcase-${SHOWCASE_VERSION}-linux-amd64.tar.gz
           cd /usr/src/showcase/
           tar -xf showcase-*
           ./gapic-showcase run &
           cd -
+        env:
+          SHOWCASE_VERSION: 0.11.0
```

Comment:
Should this be 0.16.0 too? Below as well.

Or I can do it when I rebase the pending #918

Reply:
This is a deliberate attempted rollback to see if bumping the showcase version solved anything.